discovery/openstack/mock_test.go

```diff
@@ -49,15 +51,12 @@ func (m *SDMock) Setup() {
 const tokenID = "cbc36478b0bd8e67e89469c7749d4127"
 
 func testMethod(t *testing.T, r *http.Request, expected string) {
-	if expected != r.Method {
-		t.Errorf("Request method = %v, expected %v", r.Method, expected)
-	}
+	require.Equal(t, expected, r.Method, "Expected request method.")
```

Comment:
```suggestion
	require.Equal(t, expected, r.Method, "Unexpected request method.")
```

Since we're checking if the methods are equal, this error will happen when the method is unexpected.